### PR TITLE
Basic FizzBuzz

### PR DESCRIPTION
**Pull request by VargaRaimond**

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+for (let i = 1; i <= 100; i++) {
+    if (i % 5 == 0 && i % 3 == 0)
```
Comment (cipacda) [inline index.ts:2]: It's best practice to use {} for one line blocks as well, in the cases where it's part of multiple blocks, i.e. else/if

File: index.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+        console.log("FizzBuzz");
+    else if (i % 3 == 0)
+        console.log("Fizz");
+    else if (i % 5 == 0)
+        console.log("Buzz");
+    else
+        console.log(i);
+}
```
Comment (cipacda) [inline index.ts:10]: Looks good, does the job - extra challenge would be to use at least two functions for it :) A main one and one other - can you think which one?

**Review by cipacda — APPROVED** [review]

File: index.ts (hunk 1 above)
Comment (VargaRaimond) [reply index.ts:2]: I always used them before last semester, SO projects checkstyle destroyed me 😆

File: index.ts (hunk 2 above)
Comment (VargaRaimond) [reply index.ts:10]: Maybe build the string in a function and print in main 🤔

Comment (cipacda) [reply index.ts:10]: :)

**Pull request merged**